db/statistics.go

```diff
@@ -699,13 +699,13 @@ func WriteChartSeriesForDay(day int64) error {
 	lastSlot := int64(firstSlot) + int64(epochsPerDay*utils.Config.Chain.Config.SlotsPerEpoch)
 	lastEpoch := lastSlot / int64(utils.Config.Chain.Config.SlotsPerEpoch)
 
-	latestDbEpoch, err := GetLatestEpoch()
+	finalizedCount, err := CountFinalizedEpochs(firstEpoch, uint64(lastEpoch))
 	if err != nil {
 		return err
 	}
 
-	if (uint64(lastSlot) / utils.Config.Chain.Config.SlotsPerEpoch) > latestDbEpoch {
-		return fmt.Errorf("delaying statistics export as epoch %v has not yet been indexed. LatestDB: %v", (uint64(lastSlot) / utils.Config.Chain.Config.SlotsPerEpoch), latestDbEpoch)
+	if finalizedCount < epochsPerDay {
+		return fmt.Errorf("delaying chart series export as not all epochs for this day are finalized. LatestDB: %v", day)
```

Comment:
**Minor:**
The use of `day` here is wrong, is it not? Maybe the last part can be removed?
```suggestion
		return fmt.Errorf("delaying chart series export as not all epochs for this day are finalized.")
```

Furthermore, you could add something like "(%d of %d)" with the parameters `finalizedCount` and `epochsPerDay`.

Reply:
fixed error